src/helpers/addDecksHelper.ts

```diff
@@ -87,9 +87,14 @@ export function addDecksFromJSON(
       addedDecksAndCards.push({ name, numberOfCards: cards.length });
     }
 
+    const version = fileContent.meta.version
+      ? fileContent.meta.version
+      : undefined;
+
```

Comment:
just a general remark: there is typescript feature for this, something like `fileContent?.meta?.version`

Reply:
updated with eb24802 &rarr; I also updated the test.json with the cli to have a uuid